src/events.ts

```diff
@@ -735,3 +735,39 @@ export interface PlaybackSpeedChangedEvent extends Event {
    */
   to: number;
 }
+
+/**
+ * Emitted when a subtitle entry transitions into the active status.
+ */
+export interface CueEnterEvent extends Event {
+  /**
+   * The playback time in seconds when the subtitle should be rendered.
+   */
+  start: number;
+  /**
+   * The playback time in seconds when the subtitle should be hidden.
+   */
+  end: number;
+  /**
+   * The textual content of this subtitle.
+   */
+  text?: string;
```

Comment:
Does it make sense to have the text as optional? What is the use case for a cue without text?